test/browser.transforms.test.js

```diff
@@ -452,6 +456,45 @@ describe('addBody', function() {
       });
     });
   });
+  describe('with error cause', function() {
+    // Error cause was introduced in Chrome 93.
+    if (chromeMajorVersion() < 93) return;
+
+    it('should create trace_chain', function(done) {
+      var causeErr = new Error('cause error');
```

Comment:
I know all old tests use the old `var` syntax but is there a reason to continue using them as opposed to `const`/`let`?

Reply:
Just consistency. I'd rather not have a mix until we're actually switching over to `const`/`let`.